staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -14,20 +14,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// conversion-gen is a tool for auto-generating Conversion functions.
+// conversion-gen is a tool for auto-generating functions that convert
+// between internal and external types.  A general conversion task
+// involves three packages: the one containing the internal types, the
+// one containing the external types, and the destination (i.e., the
+// one containing the conversion functions).
 //
-// Given a list of input directories, it will scan for "peer" packages and
```

Comment:
Could you also update the comments in Makefile.generated_files that mentions the "peer" packages?

IIUC, peer packages is the package with the internal types.

Reply:
Hmm, the conversion-gen also has `--base-peer-dirs` and `--extra-peer-dirs` flags. Can we keep the terms consistent?

Reply:
Updated to discuss the "peer" role and the effects of those flags.